client/components/Forms/AddServerForm.vue

```diff
@@ -58,14 +58,17 @@ export default Vue.extend({
       previousServerLength: 0
     };
   },
+  computed: {
+    ...mapState('servers', ['serverList'])
+  },
```

Comment:
What happened to the nice TypeScript mapState we were talking about ages ago?

Reply:
This is another project I discovered while working in this PR: https://github.com/maoberlehner/vuex-map-fields

Right now we can't use ``get/set`` computed properties as well, which leaves the ``DraggableQueue`` component with the old way of doing things.

iirc all the typescript stuff will be sorted out in Vuex 4

Reply:
I've been looking further for this and I confirm it's right around the corner:

https://github.com/vuejs/vuex/releases/tag/v4.0.0-beta.1

Reply:
Not sure that'll work with Vue 2, though. It mentions pretty specifically Vue 3 as the main thing.

Reply:
Yeah, I was talking specifically about when we switch to Vue 3, in fact I added it to the comments of the ``this.currentItemIndex as unknown as number`` thing.

It's a long way to go still but imo not worth it to do an overhaul now when we will need to do it later with Vuex as well, just for the same purpose.